benchmark_runner/common/clouds/IBM/assisted_installer_latest_version.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+# import urllib library
+from urllib.request import urlopen
+
+
+class OCPVersions:
+    """
+    This class get the latest assisted installer version from https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/graph
+    """
+    # MAJOR.MINOR.PATCH.BUILD
+    MAJOR = 0
+    MINOR = 1
+    PATCH = 2
+    BUILD = 3
+
+    def __init__(self):
+        self.__url = "https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/graph"
+
+    def __get_versions(self):
+        """
+        This method get the OCP versions from url and return json data
+        :return: versions in json
+        """
+        # store the response of URL
+        response = urlopen(self.__url)
```

Comment:
Check that this action succeeds and fail out with a useful message if it doesn't.